src/acceptance/java/io/codurance/pouch/GetResourceFeature.java

```diff
@@ -0,0 +1,81 @@
+package io.codurance.pouch;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.JSONException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static io.restassured.RestAssured.when;
+import static io.restassured.http.ContentType.JSON;
+import static org.apache.http.HttpStatus.SC_BAD_GATEWAY;
+import static org.apache.http.HttpStatus.SC_OK;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = PouchApiApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT)
+public class GetResourceFeature {
+
+    @LocalServerPort
+    private int serverPort;
+
+    @Value("${spring.datasource.jdbc-url}")
+    private String jdbcUrl;
+
+    private DatabaseHelper databaseHelper;
+
+    @Before
+    public void setUp() {
+        databaseHelper = new DatabaseHelper(jdbcUrl);
+        databaseHelper.clearResources();
+
+        RestAssuredConfiguration.configure(serverPort);
+    }
+
+    @Test
+    public void shouldGetOneResource() throws JsonProcessingException, JSONException {
+        var resourceOne = new DatabaseHelper.Resource(
+                1,
+                "2018-12-05 16:01:00.0",
+                "Spring Data JDBC",
+                "https://spring.io/projects/spring-data-jdbc");
+
+        databaseHelper.insertResource(resourceOne);
+
+        when().get("/resources/1")
+                .then()
+                .statusCode(SC_OK)
+                .contentType(JSON);
+
+        var actual = when().get("/resources/1")
+                .getBody()
+                .asString();
```

Comment:
Figure out how to use REST Assured to examine/assert the payload integrity.

Reply:
✅